Makefile

```diff
@@ -11,9 +11,8 @@ ifneq (, $(shell which go))
 FOUND_GO_VERSION := $(shell go version | cut -d" " -f3 | cut -d"o" -f 2)
 CORRECT_GO_VERSION := $(shell expr `go version | cut -d" " -f3 | cut -d"o" -f 2` \>= ${REQUIRED_GO_VERSION})
 endif
-TAG_COMMIT   ?= $(shell git rev-list --tags --max-count=1)
-RELEASE      ?= $(shell git describe --tags $(TAG_COMMIT))
-VERSION      ?= $(shell expr `date +%Y.%m.%d%H | cut -c 3-`)
+RELEASE      ?= 1
```

Comment:
checked this changes locally,
Previously it shows as 
`harvest version 24.03.0712-v24.02.0 (commit 07fbf34f) (build date 2024-03-07T12:03:29+0530) darwin/amd64` 
now it would changed to 
`harvest version 24.03.07-1 (commit 0d80b9c5) (build date 2024-03-07T12:05:42+0530) darwin/amd64
`

Seems fine for me